consolidated_test_report

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env sh
```

Comment:
I would move this file into the `.bin` folder, that is a good place to keep all of your executable scripts. I might also rename this to be `consolidate_test_report` so that it is more action-oriented. You could also add this to the `kill-selenium` script, but if you want to keep it separate that is also fine.

Reply:
Done. Just cant add to kill-selenium. "consolidate_test_report" file will be exported to user local machine as part zip folder. User can then just double click on the file and it starts allure server and displays results in browser.